llama.h

```diff
@@ -783,6 +783,15 @@ extern "C" {
                                   char * buf,
                                int32_t   length);
 
+    /// Get antiprompts from either the provided template or the default template of the model
+    /// @param tmpl A Jinja template to use for this chat. If this is nullptr, the model’s default chat template will be used instead.
+    /// @param buf A buffer to hold the output antiprompt. Since the length of all antiprompts is known it is assumed the alloc size is atleast 22.
```

Comment:
We cannot hard-code the length like this. The convention of all functions in `llama.h` is to always have a max length for output